.github/workflows/pr-reviewer.yaml

```diff
@@ -0,0 +1,72 @@
+name: PR Review with Progress Tracking
+
+# This example demonstrates how to use the track_progress feature to get
+# visual progress tracking for PR reviews, similar to v0.x agent mode.
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  review-with-tracking:
+    if: ${{ github.event.label.name == 'AI Review' }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      id-token: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: PR Review with Progress Tracking
+        uses: anthropics/claude-code-action@v1
+        with:
+          anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
```

Comment:
@alexandrebodin is proposing each AI action on the repo has a specific key that would enable to track the cost associated with each action.